desktop-src/SecCNG/creating-a-hash-with-cng.md

```diff
@@ -8,11 +8,27 @@ ms.date: 05/31/2018
 
 # Creating a Hash with CNG
 
-A [*hash*](/windows/desktop/SecGloss/h-gly) is a one way operation that is performed on a block of data to create a unique hash value that represents the contents of the data. No matter when the hash is performed, the same [*hashing algorithm*](/windows/desktop/SecGloss/h-gly) performed on the same data will always produce the same hash value. If any of the data changes, the hash value will change appropriately.
-
-Hashes are not useful for encrypting data because they are not intended to be used to reproduce the original data from the hash value. Hashes are most useful to verify the integrity of the data when used with an asymmetric signing algorithm. For example, if you hashed a text message, signed the hash, and included the signed hash value with the original message, the recipient could verify the signed hash, create the hash value for the received message, and then compare this hash value with the signed hash value included with the original message. If the two hash values are identical, the recipient can be reasonably sure that the original message has not been modified.
-
-The size of the hash value is fixed for a particular hashing algorithm. What this means is that no matter how large or small the data block is, the hash value will always be the same size. As an example, the SHA256 hashing algorithm has a hash value size of 256 bits.
+A [*hash*](/windows/desktop/SecGloss/h-gly) is a one way operation that is
+performed on a block of data to create a unique hash value that represents the
+contents of the data. No matter when the hash is performed, the same
+[*hashing algorithm*](/windows/desktop/SecGloss/h-gly) performed on the same
+data will always produce the same hash value. If any of the data changes, the
+hash value will change appropriately.
+
+Hashes are not useful for encrypting data because they are not intended to be
+used to reproduce the original data from the hash value. Hashes are most useful
+to verify the integrity of the data when used with an asymmetric signing
+algorithm. For example, if you hashed a text message, signed the hash, and
+included the signed hash value with the original message, the recipient could
+verify the signed hash, create the hash value for the received message, and
+then compare this hash value with the signed hash value included with the
+original message. If the two hash values are identical, the recipient can be
+reasonably sure that the original message has not been modified.
+
+The size of the hash value is fixed for a particular hashing algorithm. What
+this means is that no matter how large or small the data block is, the hash
+value will always be the same size. As an example, the SHA256 hashing algorithm
+has a hash value size of 256 bits.
```

Comment:
This is a regression for anyone willing to make changes to this content in the future. Now they have to make their changes *and* restore the intended formatting. I'd reject this change, and delegate the job of turning long lines readable to a (code) editor.